lib/index_finder.rb

```diff
@@ -0,0 +1,20 @@
+# Hides some of the dirtyness in selecting/generating the correct index objects
+class IndexFinder
```

Comment:
Not particularly - this could just as easily be a module that uses `extend self` or `module_function`. 

Some reckon that cases like this should always be a module because it's not being instantiated, and it hints to the reader that it's never meant to be. Can't say I'm particularly troubled either way.